src/main/java/com/onaple/itemizer/Itemizer.java

```diff
@@ -274,23 +273,22 @@ private void loadGlobalConfig() {
 
     public int loadItems() throws Exception {
         initDefaultConfig("items.conf");
-        return configurationHandler.readItemsConfiguration(
-                configurationHandler.loadConfiguration(configDir + "/itemizer/items.conf"));
+        return configurationHandler.readItemsConfiguration(configDir + "/itemizer/items.conf");
     }
 
     public int loadMiners() throws Exception {
         initDefaultConfig("miners.conf");
-        return configurationHandler.readMinerConfiguration(configurationHandler.loadConfiguration(configDir + "/itemizer/miners.conf"));
+        return configurationHandler.readMinerConfiguration(configDir + "/itemizer/miners.conf");
     }
 
     public int loadPools() throws Exception {
         initDefaultConfig("pools.conf");
-        return configurationHandler.readPoolsConfiguration(configurationHandler.loadConfiguration(configDir + "/itemizer/pools.conf"));
+        return configurationHandler.readPoolsConfiguration(configDir + "/itemizer/pools.conf");
     }
 
     public int loadCrafts() throws Exception {
         initDefaultConfig("crafts.conf");
-        return configurationHandler.readCraftConfiguration(configurationHandler.loadConfiguration(configDir + "/itemizer/crafts.conf"));
+        return configurationHandler.readCraftConfiguration(configDir + "/itemizer/crafts.conf");
```

Comment:
A conversion matter, I supposed you are working on it.  courage, crafts are realy complexes.